portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -25,8 +26,10 @@
 public class DataServlet extends HttpServlet {
 
   @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String text = request.getReader().lines().collect(Collectors.joining(System.lineSeparator()));
+    String name = text.substring(text.indexOf(":")+2, text.lastIndexOf("\""));
```

Comment:
Please comment on what this is parsing. What is the name expected to look like?

Reply:
I had this before using GSON to generate JSON with Java - fixed in db5f6e7d2254414a34fc04c652bdb21abb9d2e3a